holoviews/plotting/plot.py

```diff
@@ -1493,6 +1501,92 @@ def _format_title_components(self, key, dimensions=True, separator='\n'):
 
         return (label, group, type_name, dim_title)
 
+    def _parse_backend_opt(self, opt, plot, model_accessor_aliases):
+        """
+        Parses a custom option of the form 'model.accessor.option'
+        and returns the corresponding model and accessor.
+        """
+        accessors = opt.split('.')
+        if len(accessors) < 2:
+            self.param.warning("Custom option %r expects at least "
+                                "two accessors separated by '.'" % opt)
+            return
+
+        model_accessor = accessors[0]
+
+        # convert alias to handle key (figure -> fig)
+        model_accessor = model_accessor_aliases.get(model_accessor) or model_accessor
+
+        if model_accessor in self.handles:
+            model = self.handles[model_accessor]
+        elif hasattr(plot, model_accessor):
+            model = getattr(plot, model_accessor)
+        else:
+            self.param.warning("{} model could not be resolved on {} plot. "
+                                "Ensure the '{}' custom option spec "
+                                "references a valid model in the "
+                                "plot.handles {} or on the underlying "
+                                "figure object.".format(model_accessor,
+                                                    type(self).__name__,
+                                                    opt, list(self.handles.keys())))
+            return
+
+        for acc in accessors[1:-1]:
+
+            # the logic handles resolving something like:
+            # legend.get_texts()[0].set_fontsize
+            if '[' in acc and acc.endswith(']'):
```

Comment:
What if there are multiple getitems? Seems rare but possible in theory.

Reply:
Added support for `"legend.get_texts()[3,4].fontsize": 388,`

Reply:
How does it handle something like `"legend.get_texts()[3][4].fontsize"`?

Reply:
Before adding even more complexity, can you think of a valid use case for that?

I don't think this is valid:
legend.get_texts()[1][2].fontsize

legend.get_texts() => ["a label", "b label"]
[1] => "a label"
[2] => "l"
fontsize => raise AttributeError (because l is `str`)

Reply:
It will likely fail